### ISSUE TITLE
-silent flag returning just blank lines instead removing the blank lines

### ISSUE THREAD

**Issue by rudSarkar** [pwnesia/dnstake#22]
Hi @pwnesia ,

This tool is pretty good. When I use `-silent` it removed the banner and doesn't show it's fine but it returns lots of blank lines it's the same count as the "has no nameserver record" error when I don't use the `-silent` flag.

### Using `-silent` flag

![using -silent flag](https://i.imgur.com/Xad2gos.png)

### Not using `-silent` flag

![not using silent flag](https://i.imgur.com/BaVwkHo.png)

I don't understand Golang very well so I can't create a PR for this project. I hope you'll fix very soon.

Thanks,
@rudSarkar

**Comment by dwisiswant0** [comment]
Because this is how it works. That flag will only display vulnerable hosts.

**Comment by rudSarkar** [comment]
I know but it's returning just blank spaces, This is what I was talking about @dwisiswant0

**Comment by dwisiswant0** [comment]
Then your target hosts isn't vulnerable. As simple as dat.

**Comment by rudSarkar** [comment]
Hi @dwisiswant0 ,

Did you read the issue clearly? let me explain again It returns a new blank line for example:

when we use @projectdiscovery's `httpx` with `-silent` flag it doesn't return new blank lines if all the domains are not resolvable right?

But in your tools instead of showing nothing, it returns a hell lotta blank lines. if you've seen the first image clearly.

```bash
rudra:~/ $ httpx -l doms.txt -silent
rudra:~/ $
```

See there is no new blank line, below your tools with subfinder parallel and it's has space.

```bash
rudra:~/ $ subfinder -d example.com -silent | dnstake -silent




















```

I hope now you understand and set this issue label as feedback if possible.

Thanks,
@rudSarkar

**Comment by dwisiswant0** [comment]
Guess I had a misperception. Sorry & thanks for your report, will fix this asap.

**Comment by rudSarkar** [comment]
No problem. 😉

**Comment by FalcoXYZ** [comment]
Should be fixed (#24 ) @rudSarkar
